sdk/identity/azure-identity/src/main/java/com/azure/identity/ChainedTokenCredentialBuilder.java

```diff
@@ -11,11 +11,7 @@
 import java.util.Deque;
 
 /**
- * Fluent credential builder for instantiating a {@link ChainedTokenCredential}.
- *
- * The {@link ChainedTokenCredential} is a convenience credential that allows users to chain together a set of
- * TokenCredential together. The credential executes each credential in the chain sequentially and returns the token
- * from the first credential in the chain that successfully authenticates.
+ * <p>Fluent credential builder for instantiating {@link ChainedTokenCredential}.</p>
```

Comment:
Can we link the doc here as you did for DAC?

Reply:
My thought was we'd link to the doc from the credential itself, not the builder. Does that make sense?

Reply:
Ah, sure. I thought DAC had a link in both places, my mistake.